### DISCUSSION TITLE
Issue with 401

### DISCUSSION THREAD

**Discussion by spilote** [trakt/trakt-api#266] [Q&A] [answered]
I need your help guys, I was successful GET api requests for could of days. at some point it stoped. I tried remove the app from my Trakt; get a new code, token but I keep getting a 401 back. my app is registered in my personal trakt.... what am I missing here.

pi@raspberry:/var/log/openhab2 $ curl --include --request POST --header "Content-Type: application/json" --data-binary "{"code": "CODE", "client_id": "CLIENTID","client_secret": "CLIENTSECRET"}" 'https://api.trakt.tv/oauth/device/token'
HTTP/1.1 200 OK
Date: Wed, 17 Feb 2021 15:30:49 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Connection: keep-alive
Set-Cookie: __cfduid=d61812da3f8736c85a56ad3697c1eb9091613575849; expires=Fri, 19-Mar-21 15:30:49 GMT; path=/; domain=.trakt.tv; HttpOnly; SameSite=Lax
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Download-Options: noopen
X-Permitted-Cross-Domain-Policies: none
Referrer-Policy: strict-origin-when-cross-origin
Vary: Accept-Encoding
ETag: W/"d1bcb0c30478fea108b91a80ce60c7bd"
Cache-Control: max-age=0, private, must-revalidate
X-Request-Id: 601f2c55-71a2-4f1c-aa0b-6704518ba34a
X-Runtime: 0.009663
CF-Cache-Status: DYNAMIC
cf-request-id: 0852369fbb00004bb9d98cd000000001
Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
CF-RAY: 62308d45ffd34bb9-YUL

{"access_token":"TOKEN!","token_type":"bearer","expires_in":7776000,"refresh_token":"REFRESHTOKEN","scope":"public","created_at":1613575833}pi@raspberry:/var/log/openhab2 $ ^C
pi@raspberry:/var/log/openhab2 $ curl --include --header "Content-Type: application/json" --header "Authorization: TOKEN" --header "trakt-api-version: 2" --header "trakt-api-key: CLIENTID" 'https://api.trakt.tv/shows/the-undoing/progress/watched'
HTTP/1.1 401 Unauthorized
Date: Wed, 17 Feb 2021 15:32:57 GMT
Content-Type: text/html
Transfer-Encoding: chunked
Connection: keep-alive
Set-Cookie: __cfduid=d441a691f05b19cc19550fc97c69a4ab61613575977; expires=Fri, 19-Mar-21 15:32:57 GMT; path=/; domain=.trakt.tv; HttpOnly; SameSite=Lax
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Download-Options: noopen
X-Permitted-Cross-Domain-Policies: none
Referrer-Policy: strict-origin-when-cross-origin
WWW-Authenticate: Bearer realm="Doorkeeper", error="invalid_token", error_description="The access token is invalid"
Cache-Control: no-cache, no-store
Vary: Accept-Encoding
X-Ratelimit: {"name":"AUTHED_API_GET_LIMIT","period":300,"limit":1000,"remaining":999,"until":"2021-02-17T15:35:00Z"}
X-Request-Id: 99e23ebe-edf5-4383-9252-288ea0394491
X-Runtime: 0.004667
CF-Cache-Status: DYNAMIC
cf-request-id: 08523893b700003ff847856000000001
Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
CF-RAY: 62309065fa933ff8-YYZ

**Comment by rectifyer** [comment, marked as answer]
The authorization is missing the **Bearer** part.

`Authorization: Bearer [token]`
